python/mxnet/_numpy_op_doc.py

```diff
@@ -173,3 +212,73 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+
+def _np_max(axis=None, keepdims=False, initial=None, out=None):
+    """
+    Return the maximum of an array or maximum along an axis.
+
+    Parameters
+    ----------
+    a : ndarray
+        Input data.
+    axis : None or int or tuple of ints, optional
+        Axis or axes along which to operate.  By default, flattened input is
+        used.
+
+        If this is a tuple of ints, the maximum is selected over multiple axes,
+        instead of a single axis or all the axes as before.
+
+    keepdims : bool, optional
+        If this is set to True, the axes which are reduced are left
+        in the result as dimensions with size one. With this option,
+        the result will broadcast correctly against the input array.
+
+        If the default value is passed, then `keepdims` will not be
+        passed through to the `amax` method of sub-classes of
+        `ndarray`, however any non-default value will be.  If the
+        sub-class' method does not implement `keepdims` any
+        exceptions will be raised.
+
+    initial :
+        Parameter initial is not supported yet, we will support it in the future.
+        now it must be None.
+
+    out : ndarray, optional
```

Comment:
Dtype should also be the same: currently we do not support auto type casting like numpy.

Reply:
resolve,

Reply:
@gyshi Please add this in the `note` area. This is different from native numpy. I would also suggest giving a note here in `out` parameter description.